src/control/common/file_utils.go

```diff
@@ -255,3 +255,25 @@ func Run(cmd string) error {
 
 	return err
 }
+
+// GetConsent scans stdin for yes/no
+func GetConsent() bool {
```

Comment:
Is file_utils the right place for this function? There are a couple other ones in file_utils that seem not really file/directory related, too.

Reply:
FWIW, I agree with Kris here. UI stuff belongs out under cmd/whatever, not in server libraries, IMO. Furthermore, I'd argue that we shouldn't be prompting like this, because it doesn't play well with scripting. Just exit with an error telling the user that they need to supply --force for destructive operations.

Reply:
I think a new file in common (something like user_utils.go) would be OK for sharing this logic. 

I'm not sure I agree that prompting is wrong - it feels like the natural flow for a human user. If someone is scripting against our tool, I would expect them to have read enough documentation to know about the --force flag. Requiring the force flag to do anything at all, even if a human is able to respond to prompts, feels a bit weird to me.

Reply:
so changes requested are to move some of the methods in common/file_utils to say common/user_utils. I would like to keep it in common so we can use it for commands in server and dmg. doesn't seem like there is a consensus with regard to prompting so I will leave as is regarding that.